### ISSUE TITLE
20_11_2014_ClaireKelley_HW4

### ISSUE THREAD

**Issue by ClaireKelley** [malecki/qmssviz#170]
Here is my plunker for HW 4. Answers to the questions are in the HW_Answers.md file 

http://plnkr.co/edit/HicqaA5oA8401waYuDs7?p=preview

**Comment by malecki** [comment]
thanks! did you figure out your circle-packing problem?

**Comment by ClaireKelley** [comment]
yep it looks like this: http://embed.plnkr.co/8FxSOqOleAaTKOLp7JAE/preview
